features/fixtures/mazerunner/build.gradle

```diff
@@ -68,7 +76,7 @@ android {
 }
 
 dependencies {
-    implementation(name:'bugsnag-android-ndk', ext:'aar')
```

Comment:
There is the possibility with this change (even considering the exclusions above) that the tests could end up running against a release version instead of the local version. Is it now required to test the artifact this way?

Reply:
That shouldn't be possible as we [prevent gradle from fetching](https://github.com/bugsnag/bugsnag-android/pull/522/files/0720ad34d7a4298e6d0965d46f6196ebea10b18c#diff-c1c6e096014df07e7dd9bed50f7d2628R5) `com.bugsnag:bugsnag-android` from both mavenCentral and jcenter. This integration method also allows us to verify that the POM isn't malformed.

Reply:
I noted the exclusions in the original question, but I can definitely see cases where artifacts are incorrectly pulled from an unexpected source, causing the tests to pass (or fail) when they should not.

Reply:
I'm not 100% clear on what cases this could happen in, did you have an example in mind?

Reply:
bad caching :)

Reply:
I don't think this is a likely scenario as:

- mazerunner will exit before the scenarios run if `publishToMavenLocal` fails
- `mavenLocal` is specified as the first repository in the order for resolving dependencies
- `jcenter` and `mavenCentral` have been configured to stop resolving the `bugsnag-android` and `bugsnag-android-ndk` artefacts

This also improves our test coverage as it verifies the POM isn't malformed, which cannot be tested by using an AAR alone. Therefore I am leaving this as-is unless there is a concrete issue with this approach